godtools/Managers/GTDataManager.swift

```diff
@@ -10,15 +10,15 @@ import Foundation
 import Alamofire
 import PromiseKit
 import Spine
-import CoreData
+import RealmSwift
 
 class GTDataManager: NSObject {
     let documentsPath: String
     let resourcesPath: String
     let bannersPath: URL
     
     let serializer = Serializer()
-    let context = NSManagedObjectContext.mr_rootSaving()
+    let realm = try! Realm()
```

Comment:
This is opening the database (and possibly initializing it. The best case scenario here is to show the user an error message. There is no real other behavior I can think of if we cannot access the database.